PowerEditor/src/ScitillaComponent/ScintillaEditView.cpp

```diff
@@ -745,6 +745,8 @@ void ScintillaEditView::setEmbeddedAspLexer()
 		keywordList = wstring2string(kwlW, CP_ACP);
 	}
 
+	execute(SCI_SETPROPERTY, reinterpret_cast<WPARAM>("asp.default.language"), reinterpret_cast<LPARAM>("2"));
+
 	execute(SCI_SETKEYWORDS, 2, reinterpret_cast<LPARAM>(getCompleteKeywordList(keywordList, L_VB, LANG_INDEX_INSTR)));

```

Comment:
It looks better what I have tried to fix in SciLexer.dll.
However what's the meaning of value "2" on "asp.default.language" for `SCI_SETPROPERTY` ?

Reply:
2 is `eScriptVBS` in LexHTML.cxx

https://github.com/notepad-plus-plus/notepad-plus-plus/blob/1a76bd591370cf86a3b041d7ceb5bf534cc4d9b8/scintilla/lexers/LexHTML.cxx#L38